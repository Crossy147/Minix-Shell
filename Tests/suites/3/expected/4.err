/etc/passwd: permission denied
servers/bummer: no such file or directory
/etc/passwd: permission denied
servers/bummer: no such file or directory
