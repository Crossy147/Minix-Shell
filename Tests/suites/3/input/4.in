# redirection errors

ls > /etc/passwd
cat < servers/bummer

ls > /etc/passwd
cat < servers/bummer

bin/fdcounter

