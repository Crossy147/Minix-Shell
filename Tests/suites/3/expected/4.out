3 file descriptors used.
